cmd/theatre-envconsul/main.go

```diff
@@ -193,6 +225,35 @@ func mainError(ctx context.Context, command string) (err error) {
 			return errors.Wrap(err, "failed to decode envconsul environment variables")
 		}
 
+		// For every file reference, write the value resolved by envconsul to the path in
+		// filePaths
+		for key, path := range filePaths {
+			if path == pathTBD {
+				// generate file path prefixed by key
+				tempFilePath, err := ioutil.TempFile("", fmt.Sprintf("%s-*", key))
+				if err != nil {
+					return errors.Wrap(err, fmt.Sprintf("failed to write temporary file for key %s", key))
+				}
+				path = tempFilePath.Name()
+			}
+			// ensure the path structure is available
+			os.MkdirAll(filepath.Dir(path), 0644)
```

Comment:
We're not checking the returned error from this call. It can blow up, e.g. in the case that the path exists but is a file rather than a dir.